dolphinscheduler-alert/dolphinscheduler-alert-server/src/main/java/org/apache/dolphinscheduler/alert/AlertServer.java

```diff
@@ -67,7 +70,7 @@ public static void main(String[] args) {
 
     @EventListener
     public void run(ApplicationReadyEvent readyEvent) {
-        logger.info("alert server starting...");
+        logger.info("Alert server is ready ...");
```

Comment:
This is kinda misleading, it's not ready yet here, it's only ready after `alertSenderService.start();`